CHANGELOG.md

```diff
@@ -1,6 +1,139 @@
+  `VdcComputePolicy.Update`, `VdcComputePolicy.Delete` and `AdminVdc.GetAllAssignedVdcComputePolicies` ([#504](https://github.com/vmware/go-vcloud-director/pull/504))
+* Deprecated the functions `VCDClient.AddMetadataEntryByHrefAsync`
+  and `VCDClient.AddMetadataEntryByHref` in favor of `VCDClient.AddMetadataEntryWithVisibilityByHrefAsync`
+  and `VCDClient.AddMetadataEntryWithVisibilityByHref` ([#510](https://github.com/vmware/go-vcloud-director/pull/510))
+* Deprecated the functions `VCDClient.MergeMetadataByHrefAsync`
+  and `VCDClient.MergeMetadataByHref` in favor of `VCDClient.MergeMetadataWithVisibilityByHrefAsync`
+  and `VCDClient.MergeMetadataWithVisibilityByHref` ([#510](https://github.com/vmware/go-vcloud-director/pull/510))
+* Deprecated the functions `AddMetadataEntryAsync` and `AddMetadataEntry` from the following entities:
+  `VM`, `Vdc`, `AdminVdc`, `ProviderVdc`, `VApp`, `VAppTemplate`, `MediaRecord`, `Media`, `AdminCatalog`, `AdminOrg`, `Disk`,
+  `OrgVDCNetwork`, `CatalogItem` in favor of their `AddMetadataEntryWithVisibilityAsync` and `AddMetadataEntryWithVisibility`
+  counterparts ([#510](https://github.com/vmware/go-vcloud-director/pull/510))
+* Deprecated the functions `MergeMetadataAsync` and `MergeMetadataAsync` from the following entities:
+  `VM`, `Vdc`, `AdminVdc`, `ProviderVdc`, `VApp`, `VAppTemplate`, `MediaRecord`, `Media`, `AdminCatalog`, `AdminOrg`, `Disk`,
+  `OrgVDCNetwork`, `CatalogItem` in favor of their `MergeMetadataWithMetadataValuesAsync` and `MergeMetadataWithMetadataValues`
+  counterparts ([#510](https://github.com/vmware/go-vcloud-director/pull/510))
+* Deprecated the functions `DeleteMetadata` and `DeleteMetadataAsync` from the following entities:
+  `VM`, `Vdc`, `AdminVdc`, `ProviderVdc`, `VApp`, `VAppTemplate`, `MediaRecord`, `Media`, `AdminCatalog`, `AdminOrg`, `Disk`,
+  `OrgVDCNetwork`, `CatalogItem` in favor of their `DeleteMetadataWithDomainAsync` and `DeleteMetadataWithDomain`
+  counterparts ([#510](https://github.com/vmware/go-vcloud-director/pull/510))
+
+## NOTES
+* Ran `make fmt` using Go 1.19 release (`fmt` automatically changes doc comment structure). This
+  will prevent `make static` errors when running tests in pipeline using Go 1.19 ([#497](https://github.com/vmware/go-vcloud-director/pull/497))
+* Updated branding `vCloud Director` -> `VMware Cloud Director` ([#497](https://github.com/vmware/go-vcloud-director/pull/497))
+* Go officially supports 2 last releases. With Go 1.19 being released it means that Go 1.18 is the
+  minimum officially supported Go version and this set our hands free to use generics in this SDK
+  (if there is a need for it). `go.mod` is updated to reflect Go minimum version 1.18 ([#497](https://github.com/vmware/go-vcloud-director/pull/497))
+* package `io/ioutil` is deprecated as of Go 1.16. `staticcheck` started complaining about usage of
+  deprecated packages. As a result this PR switches packages to either `io` or `os` (still the same
```

Comment:
Should we omit saying "As a result this PR switches"? To something like

```suggestion
  deprecated packages. As a result packages `io` or `os` are used (still the same
```

Reply:
Fixed